packages/xo-server-audit/src/index.js

```diff
@@ -66,10 +67,8 @@ class AuditXoPlugin {
         await this._xo.getStore(NAMESPACE)
       ))
       this._auditCore = new AuditCore(storage)
-      this._blockedList = (
-        await appConf.load('xo-server-audit', {
-          appDir: path.join(__dirname, '..'),
-        })
+      this._blockedList = TOML.parse(
+        await readFile(path.join(__dirname, '..', 'config.toml'), 'utf8')
```

Comment:
What the point of this file instead of putting the list directly in the code?

Reply:
To let it configurable. If a user want to log a method which we decide to not log it.

Reply:
So, the idea is to put in the configuration all what can be configurable.

Reply:
It's not configurable, it's part of the code and will be overwritten on upgrade.

Reply:
It should be configurable no?